galvan/src/main/java/org/terracotta/testing/master/InlineServerProcess.java

```diff
@@ -316,6 +316,11 @@ public void run() {
           returnValue = false;
         }
       }
+      try {
+        stdout.close();
```

Comment:
shouldn't it be a try-resource around the while loop instead ?

Reply:
done